backend/src/main/java/de/roman/toolio/model/InquiryPart.java

```diff
@@ -12,8 +12,10 @@
 @Builder
 @Document(collection="inquiryParts")
 public class InquiryPart {
-    // Specification for the user card
+
     @Id
+    private String uuid;
+    // Specification for the user card
```

Comment:
Why not simply call it id?

I don't like the mention of part in every other variable name, mixing pre- and postfix. Think partName and then lengthPart. Imho part is not needed in the name. 

Please remove the comments, they add no real value not already gained by good variable naming. Maybe refactor to a dimension class if you feel the need to group them together?

Reply:
Something I noticed during coding as well. Will tackle this problem in a future commit, thanks for the input!